### PR TITLE
Insight template

### PR DESCRIPTION
**Pull request by froggypaule**
Hello all,

No, I am not sure this is the right way to do this: actually there are here several different branches to merge; so merging them into the one master doesn't seem right. 
Must talk to you.

### CODE REVIEW

**Comment by MarekSuchanek** [comment]
It looks great 👏🏻  Thanks!

Could use update it based on the last updates in `master` here?
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

I will then squash-merge it and update also to the latest version.

This also supersedes #3, #4, and #5, right?

**Comment by froggypaule** [comment]
Hello Marek, I did the upstream merge. Now there are 4 branches of mine; I suggest, if you agree, and after checking:
- replace your km-insight (master) with [km-insight-template-rewritten](https://github.com/froggypaule/km-insight-template/tree/km-insight-template-rewritten), simply because this as said uses the new context filter
- I would not keep [collapsible-user-friendly-km-insight-template](https://github.com/froggypaule/km-insight-template/tree/collapsible-user-friendly-km-insight-template) and [user-friendly-km-insight-template](https://github.com/froggypaule/km-insight-template/tree/user-friendly-km-insight-template) which in my opinion superseded by the [insight-template](https://github.com/froggypaule/km-insight-template/tree/insight-template) branch

I am tempted to rename this latter branch insight-template, to distinguish it from your master km-insight/km-insight-template-rewritten, to distinguish the fact that the former is meant for data stewards, and the other for developers. 
But I leave you the choice of a name... not sure ;)

So, in conclusion I would get rid of #3 and #5, but keeping #4: this is simply your original which imho can be needed by developers if one day we turn to programmatic creation/modification of KMs. 

Tell me if you find your way in all this, again I am not a git fork expert :(

**Comment by froggypaule** [comment]
Also, it should be clear that this #6 needs improvements (css, all collapse on/off missing, ...)
